erica/application/FreischaltCode/FreischaltCode.py

```diff
@@ -1,19 +1,36 @@
 from datetime import date
 
+from pydantic import BaseModel
+
 from erica.application.base_dto import BaseDto
 
 
 class FreischaltCodeRequestDto(BaseDto):
     idnr: str
-    date_of_birth: date
+    dob: date
+
+
+class FreischaltCodeRequestWithClientIdentifier(BaseModel):
+    payload: FreischaltCodeRequestDto
+    clientIdentifier: str
```

Comment:
I think this should be the FreischaltCodeDto and the class in line 8 should be the FreischaltCodePayload.

The payload should then be part of the Erica-domain. This would basically be the setup from #37 width

Reply:
I have just reorganized all dto and payload in accordance to your [branch](https://github.com/digitalservice4germany/erica/pull/37)